ui/analyse/src/study/relay/relayTourView.ts

```diff
@@ -131,7 +131,8 @@ const dateFormat = memoize(() =>
 
 const showDates = (dates: RelayTourDates) => {
   const rendered = dates.map(date => dateFormat()(new Date(date)));
-  if (rendered[1]) return `${rendered[0]} - ${rendered[1]}`;
+  // if the tournament only lasts one day, don't show the end date
+  if (rendered[1] && rendered[0] != rendered[1]) return `${rendered[0]} - ${rendered[1]}`;
```

Comment:
this should never happen because the check is done on server-side

https://github.com/lichess-org/lila/blob/377a16aa84e4063c4b8bfe839cf53fd5c4c44a98/modules/relay/src/main/RelayApi.scala#L129-L132

did you see it fail?

Reply:
yes https://github.com/lichess-org/lila/issues/15649

Reply:
The check shouldn't be done on the server side, but on the client due to timezones. You can have a tournament that span one day in UTC, but covers two days locally.